html/changelogs/geeves-stationbound_nerf.yml

```diff
@@ -0,0 +1,7 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - tweak: "Stationbounds can no longer bolt or electrify airlocks, only unbolt or de-electrify on a five second timer. This restriction does not apply to antagonist silicons."
```

Comment:
stationbounds is confusing, put "Stationbounds and the AI"

Reply:
At that point it'd be better to just put 'Lawed Synths and the AI' or 'Cyborgs and the AI' since Stationbounds is already an umbrella term for all lawed synths 👀